scala/spark-wordcount/README.md

```diff
@@ -60,10 +103,11 @@ While you will need a real Bigtable cluster, you can test the Spark job locally,
 if you have Spark insatlled. For testing, consider a Bigtable development
 cluster.
 
-    spark-submit --master local --class com.example.bigtable.spark.wordcount.WordCount \
-    target/cloud-bigtable-dataproc-spark-wordcount-0.1-jar-with-dependencies.jar \
-    $GOOGLE_CLOUD_PROJECT $BIGTABLE_INSTANCE $BIGTABLE_TABLE \
-    $WORDCOUNT_FILE
+    spark-submit \
+        --class com.example.bigtable.spark.wordcount.WordCount \
```

Comment:
why doesnt this need --master local anymore?

Reply:
`--master local` is implied by `spark-submit` (that uses as many CPUs cores as possible not just one).

Reply:
do you think it makes sense to keep that to be explicit?

Reply:
@kolea2 No, never. It only makes `spark-submit` more confusing with too many options. I'm also going to propose a change to remove the need for `--class com.example.bigtable.spark.wordcount.WordCount` as it can be inferred (and hopefully simpler for people new to all these tools).

Reply:
The change has been proposed in the other PR (awaiting your approval).

Reply:
I think that keeping it explicit will make it easier for users to find the entry point. The manifest mainClass approach is nice for invoking things, but adds significant amount of indirection. Lets keep the class name